core/model/src/identity.rs

```diff
@@ -211,6 +211,24 @@ impl RpcMessage for Subscribe {
     type Error = Error;
 }
 
+#[derive(Clone, Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct GetKeyFile {
```

Comment:
Can I just alias this struct (~ `type GetKeyFile = NodeId` ) just to be able to bind... :question: 
Seems like unnecessary struct declaration.

Reply:
Nope, you wouldn't be able to add any `impl` for a foreign type. The most minimal solution would be:
```rust
pub struct GetKeyFile(pub NodeId);
```
Please note that implementing `new` is not necessary if fields are `pub`.